### DISCUSSION TITLE
Povratna informacija na prvu provjeru projekta

### DISCUSSION THREAD

**Discussion by dperas** [AIR-FOI-HR/AIR2206#1] [General]
Poštovane kolegice i kolege,
evo povratnih informacija na prvu provjeru projekta:

- Arhitektura sustava treba biti izrađena do razine modula. 
- Timska dokumentacija je samo započeta, treba biti puno detaljnija. 
- Potrebno je izraditi korisničke priče i wireframe. 
- Issues i epics u Jiri nisu ispravno podešeni, te nisu definirani sprintevi. Svaki issue treba odgovarati korisničkoj priči (pogledajte upute za prvu provjeru projekta). Svakom issue treba definirati story  points i original estimate. 

Gotovo nijedan task nije dovršen, pa ste dobili minimalan broj bodova (6). 
Ako sam negdje nešto previdjela, molim da me uputite na ispravnu dokumentaciju.

Molim da za drugu provjeru projekta napravite sve što nedostaje u ovoj fazi, uključujući i taskove koje ćete dobiti za drugu provjeru projekta. 

LP,
Dijana Peras

**Comment by denis137255** [comment]
Poštovana profesorice, 

Hvala na povratnim informacijama. Sve navedeno smo uzeli u obzir te dodali u popis stvari koje treba doraditi i napraviti. Inicijalni raspored je obavljen, tako da smatramo da neće biti većih poteškoća u ispravljanju istog.

Hvala i lp,
Tim 2206
